modules/net-cloudnat/variables.tf

```diff
@@ -68,6 +64,25 @@ variable "config_timeouts" {
   nullable = false
 }
 
+variable "endpoint_types" {
+  description = "Specifies the endpoint Types supported by the NAT Gateway. Supported values include: ENDPOINT_TYPE_VM, ENDPOINT_TYPE_SWG, ENDPOINT_TYPE_MANAGED_PROXY_LB"
+  type        = list(string)
+  default     = null
+  validation {
+    condition = (var.endpoint_types == null ? true : setunion([
```

Comment:
nit: why not use `contains` in half the lines? smart way of doing it though, made me smile :)

Reply:
ignore me :)